diff --git a/src/Normalizer/NetworkSettingsNormalizer.php b/src/Normalizer/NetworkSettingsNormalizer.php
index 6d82a9e..3b0fa38 100644
--- a/generated/Normalizer/NetworkSettingsNormalizer.php
+++ b/generated/Normalizer/NetworkSettingsNormalizer.php
@@ -57,8 +57,12 @@ class NetworkSettingsNormalizer implements DenormalizerInterface, NormalizerInte
             $values = new \ArrayObject([], \ArrayObject::ARRAY_AS_PROPS);
             foreach ($data->{'Ports'} as $key => $value) {
                 $values_1 = [];
-                foreach ($value as $value_1) {
-                    $values_1[] = $this->denormalizer->denormalize($value_1, 'Docker\\API\\Model\\PortBinding', 'json', $context);
+                if (null !== $value){
+                    foreach ($value as $value_1) {
+                        $values_1[] = $this->denormalizer->denormalize($value_1, 'Docker\\API\\Model\\PortBinding', 'json', $context);
+                    }
+                } else {
+                    $values_1[] = null;
                 }
                 $values[$key] = $values_1;
             }
@@ -138,8 +142,12 @@ class NetworkSettingsNormalizer implements DenormalizerInterface, NormalizerInte
             $values = new \stdClass();
             foreach ($object->getPorts() as $key => $value) {
                 $values_1 = [];
-                foreach ($value as $value_1) {
-                    $values_1[] = $this->normalizer->normalize($value_1, 'json', $context);
+                if (null !== $value) {
+                    foreach ($value as $value_1) {
+                        $values_1[] = $this->normalizer->normalize($value_1, 'json', $context);
+                    }
+                } else {
+                    $values_1 = null;
                 }
                 $values->{$key} = $values_1;
             }